src/main/kotlin/parts/wisdom/arcraider/visualization/ARCWindow.kt

```diff
@@ -3,48 +3,57 @@ package parts.wisdom.arcraider.visualization
 import parts.wisdom.arcraider.SerializedGrid
 import parts.wisdom.arcraider.Task
 import parts.wisdom.arcraider.loadTaskFromFile
+import java.awt.BorderLayout
 import java.awt.Dimension
 import javax.swing.Box
 import javax.swing.BoxLayout
-import javax.swing.JButton
-import javax.swing.JFileChooser
 import javax.swing.JFrame
 import javax.swing.JLabel
 import javax.swing.JPanel
 import javax.swing.JToggleButton
+import kotlin.math.max
 
 private const val GAP_SIZE = 12
-private const val GRID_SIZE = 40
+private const val GRID_SIZE = 400
```

Comment:
I always advocate for including the units in names of numeric variables when not obvious in context.